charts/redpanda/Chart.yaml

```diff
@@ -23,7 +23,7 @@ type: application
 # The chart version and the app version are not the same and will not track
 # together. The chart version is a semver representation of changes to this
 # chart.
-version: 5.7.39
+version: 5.7.40
```

Comment:
nit: This will conflict with the version bump from your other PR.

Reply:
You are right. I will fix it in the next push.